common-lib/src/main/scala/com/gu/mediaservice/model/usage/PrintUsageMetadata.scala

```diff
@@ -41,13 +41,20 @@ case class PrintUsageMetadata(
   type LongElement = List[(String,Long)]
   type IntElement = List[(String,Int)]
 
+  val calculatedPublicationName =
+    if (publicationName.trim.isEmpty) publicationCode match {
+      case "gdn" => "Guardian"
+      case "obs" => "Observer"
+      case _ => "Unknown Publication"
+    } else publicationName
```

Comment:
Minor: should this also be trimmed? Or should we trim prior to storing the data?